packages/server/lib/browsers/firefox-util.ts

```diff
@@ -221,7 +221,7 @@ export default {
     marionettePort,
     foxdriverPort,
     remotePort,
-  }) {
+  }): Bluebird<[void, void, BrowserCriClient]> {
```

Comment:
Since the first two promises return null, should setup filter this in a `.then()` to only return the BrowserCriClient?

Reply:
Good call: https://github.com/cypress-io/cypress/commit/142ad0e50835b5c8ad9f1a614815ca2e040601f2